docs/structure-of-a-contract.rst

```diff
@@ -56,7 +56,8 @@ Functions are the executable units of code within a contract.
 
 :ref:`function-calls` can happen internally or externally
 and have different levels of :ref:`visibility<visibility-and-getters>`
-towards other contracts.
+towards other contracts. Functions accept :ref:`parameters and return variables<function-parameters-return-variables>` to pass parameters and
```

Comment:
I wonder if just using 'values' here would be clearer if not 100% accurate?

Reply:
Yes, values would be better.

Could you also link "Functions" above with "functions"?

Reply:
Done